src/api-reference/direct-connects/hotel-service-2/Search.markdown

```diff
@@ -292,9 +296,17 @@ The maximum allowed size of `OTA_HotelSearchRS` is 1 MB. Any response that excee
 |------------------------|-------------------|-------------|
 |`HotelPreference`|`stringLength1to32`|**Required** SAP Concur allows customers to override property preference in the system settings. Supported values: `not_preferred`, `less_preferred`, `preferred`, `most_preferred`|
 |`TPA_HotelPreviewImageURI`|`complex`|**Required** Details for an image of a given category.|
+|`TPA_PropertyReferenceInfo`|`complex`|Alternate property ID. **Required** for Corporate Discount Note support|
```

Comment:
This is overloading the typical usage of "Required" in the docs.
I propose not having it bolded, as that sets the expectation that we could reject the response if it's missing. e.g. 
"Alternate Property ID. Required for Corporate Discount Notes support"

Reply:
This now makes me wonder why the other TPA extensions are marked required

Reply:
done